sdk/src/client/node_api/core/routes.rs

```diff
@@ -129,14 +130,38 @@ impl ClientInner {
         const PATH: &str = "api/core/v3/committee";
 
         let epoch_index = epoch_index.into().map(|i| format!("epochIndex={i}"));
-
         self.node_manager
             .read()
             .await
             .get_request(PATH, epoch_index.as_deref(), self.get_timeout().await, false, false)
             .await
     }
 
+    /// Returns information of all registered validators and if they are active.
+    /// GET JSON to /api/core/v3/validators
+    pub async fn get_validators(&self, page_size: Option<u32>) -> Result<ValidatorsResponse> {
```

Comment:
page_size alone is useless, we would also need to be able to provide a cursor https://github.com/iotaledger/tips-draft/pull/24/files#r1308764062

Reply:
https://github.com/iotaledger/iota-sdk/issues/1118